content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +93,371 @@ service:
       exporters: [datadog/api]
 ```
 
+### Environment Specific Setup
+
+#### Host:
+
+- Download the appropriate binary from [the project repository latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/latest).
+
+- Create a `otel_collector_config.yaml` file. Here is an example template to get started. It enables the collector's otlp receiver and datadog exporter.
```

Comment:
"Here is an example" Should we have a link to a sample? Which sample do you have in mind?
```suggestion
- Create a `otel_collector_config.yaml` file. Here is an example template to get started. It enables the collector's `otlp` receiver and the Datadog exporter.
```
or OTLP? are you using "otlp" as a code keyword or as an adjective?

Reply:
it should be, i believe  `OTLP`  which stands for `OpenTelemetry Protocol`

- https://github.com/open-telemetry/oteps/blob/master/text/0035-opentelemetry-protocol.md#opentelemetry-protocol-specification

will update